compiler/stable_mir/src/mir/pretty.rs

```diff
@@ -58,16 +58,16 @@ pub fn pretty_statement(statement: &StatementKind) -> String {
             pretty.push_str(format!("        _{} = ", place.local).as_str());
             pretty.push_str(format!("{}", &pretty_rvalue(rval)).as_str());
         }
-        StatementKind::FakeRead(_, _) => todo!(),
-        StatementKind::SetDiscriminant { .. } => todo!(),
-        StatementKind::Deinit(_) => todo!(),
-        StatementKind::StorageLive(_) => todo!(),
-        StatementKind::StorageDead(_) => todo!(),
-        StatementKind::Retag(_, _) => todo!(),
-        StatementKind::PlaceMention(_) => todo!(),
-        StatementKind::AscribeUserType { .. } => todo!(),
-        StatementKind::Coverage(_) => todo!(),
-        StatementKind::Intrinsic(_) => todo!(),
+        StatementKind::FakeRead(_, _) => (),
+        StatementKind::SetDiscriminant { .. } => (),
+        StatementKind::Deinit(_) => (),
+        StatementKind::StorageLive(_) => (),
+        StatementKind::StorageDead(_) => (),
+        StatementKind::Retag(_, _) => (),
+        StatementKind::PlaceMention(_) => (),
+        StatementKind::AscribeUserType { .. } => (),
+        StatementKind::Coverage(_) => (),
+        StatementKind::Intrinsic(_) => (),
```

Comment:
Can you at least add a fixme comment to finish these

Reply:
Added fix me and now they return string explaining they are unimplemented